docker/cosmosdb/Dockerfile

```
@@ -0,0 +1,7 @@
+FROM node:18.13.0-alpine
```

```suggestion
FROM node:20.12.2-alpine
```
let's use the same base image when possible to further reduce pipeline time